### PR TITLE
File Manager - Input of "New folder" dialog should has default value and active state

### PR DESCRIPTION
**Pull request by maxerokh**

### CODE REVIEW

File: js/localization/messages/en.json [hunk 1]
```diff
@@ -285,6 +285,7 @@
         "dxHtmlEditor-heading": "Heading",
         "dxHtmlEditor-normalText": "Normal text",
 
+        "dxFileManager-newFolderName": "Untitled folder",
```
Comment (RomanTsukanov) [inline js/localization/messages/en.json:288]: Can't we use "New folder", like in Windows?

**Review by vladkovl — DISMISSED** [review]

File: js/ui/file_manager/ui.file_manager.dialog.name_editor.js [hunk 2]
```diff
@@ -21,6 +21,15 @@ class FileManagerNameEditorDialog extends FileManagerDialogBase {
         super.show();
     }
 
+    _onPopupShown() {
+        if(!this._nameTextBox) {
+            return;
+        }
+        const $textBoxInput = $(this._nameTextBox.element()).find("input");
```
Comment (DokaRus) [inline js/ui/file_manager/ui.file_manager.dialog.name_editor.js]:
```suggestion
        const $textBoxInput = this._nameTextBox._input();
```

File: js/localization/messages/en.json (hunk 1 above)
Comment (maxerokh) [reply js/localization/messages/en.json:288]: I have reviewed  Google Drive and Unix systems. The systems have "Untitled folder" default name for new directories. How do you think, we should use "New folder" name and follow for the Windows naming?

Comment (RomanTsukanov) [reply js/localization/messages/en.json:288]: We decided to go with "Untitled folder"

**Review by RomanTsukanov — DISMISSED** [review]

File: js/localization/messages/en.json (hunk 1 above)
Comment (maxerokh) [reply js/localization/messages/en.json:288]: We decide to use the "Untitled folder"

**Review by vladkovl — DISMISSED** [review]

**Review by vladkovl — APPROVED** [review]

File: js/localization/messages/de.json [hunk 3]
```diff
@@ -254,6 +254,7 @@
         "dxHtmlEditor-heading": "Überschrift",
         "dxHtmlEditor-normalText": "Normaler Text",
 
+        "dxFileManager-newFolderName": "TODO!",
```
Comment (BingoRUS) [inline js/localization/messages/de.json:257]: It is not cute 😑

**Pull request merged**